pycytominer/cyto_utils/util.py

```diff
@@ -176,6 +177,52 @@ def check_consensus_operation(operation: str) -> str:
     return operation
 
 
+def maybe_write_to_file(
```

Comment:
```suggestion
def write_to_file_if_user_specifies_output_details(
```

I think adding specificity here (describing the "maybe" directly) helps readability. Please feel free to modify my suggestion as well.